test/hw2/test_task2.py

```diff
@@ -0,0 +1,13 @@
+from All_home_works.hw2.task2 import major_and_minor_elem
+
+
+def test_major_and_minor_elem_example1_from_description():
```

Comment:
`example1`,  `example2` means that this tests test different cases.  So you need to put this difference into the names.  

Or you may not find any, because these tests actually do the same and one of them could be removed.